internal/rbd/mirror.go

```diff
@@ -183,6 +187,8 @@ func (rm *rbdMirror) Promote(ctx context.Context, force bool) error {
 		return fmt.Errorf("failed to promote image %q with error: %w", rm, err)
 	}
 
+	log.DebugLog(ctx, "image %q has been promoted", rm)
```

Comment:
Please make similar changes for group as well in downstream branch

Reply:
It is already there for group, so we are good there.